### ISSUE TITLE
Running cargo build from importing 0.7.3 or from `main` fails

### ISSUE THREAD

**Issue by mothran** [atlas-aero/rt-embedded-redis#12]
Running `cargo build` on `main` or when importing 0.7.3 of the crate I get the following error:
```
error[E0599]: no method named `stream_position` found for type parameter `W` in the current scope
  --> /home/user/.cargo/registry/src/index.crates.io-6f17d22bba15001f/cookie-factory-0.3.3/src/internal.rs:92:34
   |
90 | impl<W: Seek> io::Seek for WriteContext<W> {
   |      - method `stream_position` not found for this type parameter
91 |     fn seek(&mut self, pos: SeekFrom) -> io::Result<u64> {
92 |         let old_pos = self.write.stream_position()?;
   |                                  ^^^^^^^^^^^^^^^ method not found in `W`
```

**Comment by marius-meissner** [comment]
Thanks for your report. The issue will be fixed as soon #13 is done.
For the moment, you could work around by pinning the cookie-factory crate version to 0.3.2:
```Toml
cookie-factory = "=0.3.2"
```

**Comment by marius-meissner** [comment]
@mothran As #15 has been done and released in v0.8. Could you please check if the issue is solved now?

**Comment by mothran** [comment]
Confirmed it works, thanks!
